### ISSUE TITLE
[Open Data] Update public Open Data Inventory on the DTS Portal 

### ISSUE THREAD

**Issue by KaroEngstrom** [cityofaustin/atd-data-tech#10750]
This issue is about the public data inventory page [here](https://atd.knack.com/dts#open-data-inventory/). 
Another issue (https://github.com/cityofaustin/atd-data-tech/issues/10748) has been made for [the dataset inventory that requires sign-in](https://atd.knack.com/dts#datasets/). 

After the initial Open Data Clean-Up for the pending City Data Hub migration, we have updated information for the inventory of datasets [here](https://atd.knack.com/dts#open-data-inventory/). Once the City Data Hub migration is complete, I want to update this inventory to accurately reflect our public open datasets at the time. 

Estimated time for working on this: after City Data Hub migration completion, so in March/April 2023.

**Comment by KaroEngstrom** [comment]
Sun 3/5/23 

- current inventory of `instance` records where Host Application field = ODP/Socrata; count of 37 datasets
